metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -449,3 +449,15 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: InvalidStorageAutoScalerCRsFound
+      annotations:
+        description: |-
+          Invalid phase detected for StorageAutoScaler {{ $labels.name }} attached with device class {{ $labels.device_class }} under storage cluster {{ $labels.storagecluster }} in the namespace {{ $labels.namespace }}.
+        message: StorageAutoScaler is in an invalid state.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/InvalidStorageAutoScalerCRsFound.md
+        severity_level: warning
+      expr: |
+        ocs_storageautoscaler_phase_status == 5
+      for: 1m
+      labels:
+        severity: warning
```

Comment:
@parth-gr , currently severity/severity-level is in `warning` level. (As per one of your previous comments) Do you want to change it to an `info` level alert? Please let me know

Reply:
I am fine with Info level, 
cc @travisn if you have any other opinion for it